### PR TITLE
[BUG, MRG] Fix tutorial + example show_view to use correct keyword args

### PR DESCRIPTION
**Pull request by alexrockhill**
This is in reference to #9598.

Also if there were two hemispheres but the same view, it would do the depreciation warning so I fixed that.

### CODE REVIEW

**Comment by larsoner** [comment]
> Also if there were two hemispheres but the same view, it would do the depreciation warning so I fixed that.

Makes sense, to be safe can you make sure you touch/change all six failed examples in https://app.circleci.com/pipelines/github/mne-tools/mne-python/9192/workflows/843a1e8e-b046-48bc-9909-908f9b29b317/jobs/31440

File: tutorials/simulation/80_dics.py [hunk 1]
```diff
@@ -280,8 +279,7 @@ def plot_approach(power, n):
     brain.add_foci(vertices[0][0], coords_as_verts=True, hemi='lh', color='b')
     brain.add_foci(vertices[1][0], coords_as_verts=True, hemi='rh', color='b')
     # Rotate the view and add a title.
-    brain.show_view(view={'azimuth': 0, 'elevation': 0, 'distance': 550,
-                          'focalpoint': [0, 0, 0]})
+    brain.show_view(azimuth=0, elevation=0, distance=550, focalpoint=(0, 0, 0))
```
Comment (larsoner) [inline tutorials/simulation/80_dics.py:282]: What a nicer API :)

**Review by larsoner — APPROVED** [review]
LGTM, will merge once CircleCI comes back happy. Thanks for the quick fix @alexrockhill !

**Comment by alexrockhill** [comment]
@larsoner, do you know what's going on with this failed test?

**Comment by larsoner** [comment]
Looks unrelated to your changes, I'm guessing it's VTK 9.0.3's fault. I'll open a PR to try to fix it

**Comment by larsoner** [comment]
Thanks @alexrockhill !

**Pull request merged**